src/components/organization/configuration-settings/CreateDid.tsx

```diff
@@ -250,137 +242,62 @@ const CreateDIDModal = (props: EditOrgdetailsModalProps) => {
 						values: IFormikValues,
 						{ resetForm }: FormikActions<IFormikValues>,
 					) => {
-						const didMethodValue = showMethod(
-							values.method,
-							values.ledger,
-							values.method,
-							values.network,
-						);
 
-						const didMethodName = didMethodValue
-							.split(':')
-							.slice(0, 2)
-							.join(':');
-						let selectedLedgerName;
-
-						switch (didMethodName) {
-							case 'did:indy':
-								selectedLedgerName = didMethodValue
-									.split(':')
-									.slice(-2)
-									.map((word) => word.charAt(0).toUpperCase() + word.slice(1))
-									.join(' ');
-
-								break;
-
-							case 'did:polygon':
-								selectedLedgerName = didMethodValue
-									.split(':')
-									.slice(1)
-									.map((part) => part.charAt(0).toUpperCase() + part.slice(1))
-									.join(' ');
-
-								break;
-
-							case 'did:web':
-							case 'did:key':
-								selectedLedgerName = didMethodValue.split(':')[1];
-
-								break;
-
-							default:
-								console.error('Unsupported DID format');
-						}
-
-						if (ledgerName !== selectedLedgerName) {
-							setErrMsg('This ledger is not applicable to create a DID');
-						} else {
-							await createNewDid(values);
-							setErrMsg(null);
-							window.location.reload();
-						}
+						await createNewDid(values);
+						window.location.reload();
```

Comment:
Any specific reason for using reload() method

Reply:
Resolved